Applications/Tizen_CAPI/capi_file.c

```diff
@@ -102,7 +102,7 @@ int main(int argc, char *argv[]) {
   NN_RETURN_STATUS();
 
   /* delete model */
-  status = ml_nnmodel_destruct(model);
+  status = ml_train_model_destroy(model);
```

Comment:
Which one is better ? construct - destroy or construct-destruct , or create-destroy, create-destruct?

Reply:
I liked create-destroy which is also what nnstreamer API follows.